scripts/pre_commit_linter.py

```diff
@@ -949,7 +959,8 @@ def __init__(self, all_filepaths, verbose_mode_enabled=False):
         """
         self.all_filepaths = all_filepaths
         self.verbose_mode_enabled = verbose_mode_enabled
-        self.parsed_js_files = self._validate_and_parse_js_files()
+        self.parsed_js_files, self.read_js_files = (
```

Comment:
Why this change?

Reply:
This was done because `parsed_js_files` had esprima-parsed JS files whereas the regex checks for each line and the `read_js_files` reads the file line by line without parsing for the regex to work.